WinUIGallery/Pages/NavigationRootPage.xaml

```diff
@@ -271,48 +307,12 @@
                             Tag="AccessibilityColorContrast" />
                     </NavigationViewItem.MenuItems>
                 </NavigationViewItem>
-
-                <NavigationViewItem
-                    x:Name="XamlFundamentalsItem"
-                    AutomationProperties.AutomationId="XamlFundamentalsItem"
-                    Content="XAML fundamentals"
-                    Tag="XamlFundamentals">
+                <NavigationViewItemHeader Content="Controls" />
+                <NavigationViewItem x:Name="AllControlsItem" Content="All">
                     <NavigationViewItem.Icon>
-                        <PathIcon Data="M12.9365 4.05224C13.1033 3.67311 12.9312 3.23052 12.5521 3.0637C12.1729 2.89688 11.7303 3.069 11.5635 3.44813L6.06352 15.9481C5.8967 16.3273 6.06882 16.7699 6.44795 16.9367C6.82709 17.1035 7.26967 16.9314 7.43649 16.5522L12.9365 4.05224ZM14.2927 13.8447C13.9644 13.5921 13.903 13.1212 14.1555 12.7929L16.3038 10.0002L14.1555 7.20749C13.903 6.87917 13.9644 6.40829 14.2927 6.15574C14.621 5.90319 15.0919 5.96461 15.3445 6.29292L17.8445 9.54292C18.0518 9.81251 18.0518 10.1879 17.8445 10.4575L15.3445 13.7075C15.0919 14.0358 14.621 14.0972 14.2927 13.8447ZM5.70728 6.15581C6.0356 6.40836 6.09702 6.87925 5.84447 7.20756L3.69622 10.0003L5.84447 12.793C6.09702 13.1213 6.0356 13.5922 5.70728 13.8447C5.37897 14.0973 4.90808 14.0359 4.65553 13.7076L2.15553 10.4576C1.94816 10.188 1.94816 9.81259 2.15553 9.543L4.65553 6.293C4.90808 5.96468 5.37897 5.90326 5.70728 6.15581Z" />
+                        <FontIcon Glyph="&#xF0E2;" />
```

Comment:
```suggestion
                        <FontIcon Glyph="&#xE8A9;" />
```
Visually identical, but F0E2 is called "GridView" and this one is called "ViewAll" 🤓

Reply:
Fixed! Thanks!